webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './StoppedTasksCard.scss';
+import {
+  getTableItemClassName,
+  shouleBeSelected,
+} from './StoppedTasksCardHelpers';
+
+const StoppedTasksCard = ({
+  className,
+  error,
+  warning,
+  success,
+  timePeriod,
+  focusedOn,
+  onTitleClick,
+}) => {
+  const data = { error, warning, success };
+
+  return (
+    <Card
+      className={classNames(
+        'tasks-donut-card',
+        'stopped-tasks-card',
+        className,
+        {
+          'selected-tasks-card': shouleBeSelected(focusedOn),
+        }
+      )}
+    >
+      <Card.Title onClick={onTitleClick}>{__('Stopped')}</Card.Title>
+      <Card.Body>
+        <table className="table table-titles">
+          <tbody>
+            <tr>
+              <td />
+              <td>Total</td>
+              <td>{timePeriod}</td>
+            </tr>
+          </tbody>
+        </table>
+        <table className="table table-bordered table-striped">
+          <tbody>
+            {Object.keys(data).map((item, index) => (
+              <tr className={`${item}-row`} key={index}>
+                <td>{item[0].toUpperCase() + item.slice(1)}</td>
+                <td
+                  className={getTableItemClassName(item, 'total', focusedOn)}
+                  onClick={data[item].total.onClick}
+                >
+                  {data[item].total.value}
+                </td>
+                <td
+                  className={getTableItemClassName(item, 'last', focusedOn)}
+                  onClick={data[item].last.onClick}
+                >
+                  {data[item].last.value}
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </Card.Body>
+    </Card>
+  );
+};
+
+const focusedOnResultPropType = PropTypes.shape({
+  total: PropTypes.bool,
+  last: PropTypes.bool,
+});
+
+const resultPropType = PropTypes.shape({
+  total: PropTypes.shape({
```

Comment:
This shape can be reused as well

Reply:
Make everything `required` here can help when reading the data. (won't need to do `warrning.total && warning.total.value`)

Reply:
Already required, and I couldn't find a place that uses `warning.total && warning.total.value`